lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/visitor.dart

```diff
@@ -5,6 +5,9 @@ const commentsOperator = {
   _CommentType.documentation: '///',
 };
 
+final _regTemplateExp = RegExp(r"{@template [\w'-]+}");
+final _regMacroExp = RegExp(r"{@macro [\w'-]+}");
+
```

Comment:
Most of the example I've seen online also uses `.` in the template names. And this is supported by dartdoc.

Wouldn't this be better?

```suggestion
final _regTemplateExp = RegExp(r"{@template [\w'-\.]+}");
final _regMacroExp = RegExp(r"{@macro [\w'-\.]+}");

```

ex: 

```dart
/// {@template my_project.my_class.my_method}
```

Reply:
In general, it can even be combined and simplified to `r'{@(template|macro) .+}'`.

I don't see any actual profit in making it stricter.

Reply:
Thanks! Done))